plugins/xen/xen-multi

```diff
@@ -146,13 +151,16 @@ while (<XENTOP>) {
 
   # We need the remaining data only for a normal run
   if ($ARGV[0] eq "") {
-    $domains{$domain}{'cpusecs'} = $cpusecs;
-    $domains{$domain}{'cpupercent'} = $cpupercent;
-    $domains{$domain}{'mem'} = $memk;
-    $domains{$domain}{'nettx'} = $nettxk;
-    $domains{$domain}{'netrx'} = $netrxk;
-    $domains{$domain}{'vbdrd'} = $vbdrd;
-    $domains{$domain}{'vbdwr'} = $vbdwr;
+    $domains{$domain}{'cnt'}++;
+    if ( $domains{$domain}{'cnt'} > 1 && ( !defined $domains{$domain}{'cpusecs'} || $cpusecs < $domains{$domain}{'cpusecs'} ) ) {
```

Comment:
Please add a comment here, explaining this logic. Or maybe just move the "4 iterations" comment from above down to this line?

Reply:
Fixed